### PR TITLE
data: njogued

### PR DESCRIPTION
**Pull request by njogued**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/njogued.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Edward Njogu",
+  "avatar": "https://github.com/njogued.png",
+  "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/njogued.json:4]: these are no longer required:

```suggestion
```

File: data/njogued.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Edward Njogu",
+  "bio": "Fullstack Developer",
+  "tags": [
+    "Javascript",
+    "Python",
+    "Django",
+    "Bootstrap",
+    "Full-stack",
```
Comment (SaraJaoude) [inline data/njogued.json:9]: removing comma:
```suggestion
    "Full-stack"
```

File: data/njogued.json [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    "Python",
+    "Django",
+    "Bootstrap",
+    "Full-stack"
+  ],
+  "socials": [
+    { "icon": "FaGlobe", "url": "https://njogued.me" },
+    { "icon": "FaGithub", "url": "https://github.com/njogued" },
+    { "icon": "FaMedium", "url": "https://medium.com/@njogued" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/edward-njogu-36b345122/" },
```
Comment (SaraJaoude) [inline data/njogued.json:15]: removing comma:
```suggestion
    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/edward-njogu-36b345122/" }
```

File: data/njogued.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  ],
+  "links": [{
+      "name": "Portfolio Website",
+      "url": "http://njogued.me",
+      "icon": "FaGlobe"
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/njogued",
+      "icon": "FaGithub",
```
Comment (SaraJaoude) [inline data/njogued.json:25]: removing comma:
```suggestion
      "icon": "FaGithub"
```

File: data/njogued.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    {
+      "name": "Github",
+      "url": "https://github.com/njogued",
+      "icon": "FaGithub"
+    },  
+    {
+      "name": "LinkedIn: Follow me",
+      "url": "https://www.linkedin.com/in/edward-njogu-36b345122/",
+      "icon": "FaLinkedin",
```
Comment (SaraJaoude) [inline data/njogued.json:30]: removing comma:
```suggestion
      "icon": "FaLinkedin"
```

File: data/njogued.json [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    },  
+    {
+      "name": "LinkedIn: Follow me",
+      "url": "https://www.linkedin.com/in/edward-njogu-36b345122/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Medium",
+      "url": "https://medium.com/@njogued",
+      "icon": "FaMedium",
```
Comment (SaraJaoude) [inline data/njogued.json:35]: removing comma:
```suggestion
      "icon": "FaMedium"
```

File: data/njogued.json [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+      "name": "LinkedIn: Follow me",
+      "url": "https://www.linkedin.com/in/edward-njogu-36b345122/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Medium",
+      "url": "https://medium.com/@njogued",
+      "icon": "FaMedium"
+    }
+  ],
```
Comment (SaraJaoude) [inline data/njogued.json:37]: removing comma:
```suggestion
  ]
```

**Review by SaraJaoude — APPROVED** [review]
I have made the necessary changes for the Json to pass and I will now merge this PR.

**Comment by njogued** [comment]
Thanks Sara!

**Pull request merged**